packages/core/src/telemetry/constants.ts

```diff
@@ -32,6 +34,24 @@ export const METRIC_API_REQUEST_LATENCY = 'gemini_cli.api.request.latency';
 export const METRIC_TOKEN_USAGE = 'gemini_cli.token.usage';
 export const METRIC_SESSION_COUNT = 'gemini_cli.session.count';
 export const METRIC_FILE_OPERATION_COUNT = 'gemini_cli.file.operation.count';
+
+export const FEEDBACK_CONTENT_MAX_LENGTH = 4096;
```

Comment:
Let's remove all the remnants of textual feedback. (including this const, the event above and the truncate func here)